docs/DeveloperGuide.md

```diff
@@ -312,50 +312,127 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
 ### Use cases
 
-(For all use cases below, the **System** is the `AddressBook` and the **Actor** is the `user`, unless specified otherwise)
+(For all use cases below, the **System** is `TutorMate` and the **Actor** is the `Tutor`, unless specified otherwise)
+
+### Use case: Install the app [UC01]
 
-**Use case: Install the app**
 MSS 
-1. User execute the installation package
-2. The app ask for continue or cancel the installation
-3. User respond by indicating continue
-4. The app install the app
+1. User executes the installation package
+2. The app asks for continue or cancel the installation
+3. User responds by indicating continue
+4. The app installs the app
 5. The app informs the user that the installation is successful
 Use Case ends
 
 Extensions
-* 3a. User respond by indicating cancel
-  * 3a1. The app close the installation package
+* 3a. User responds by indicating cancel
+  * 3a1. The app closes the installation package
   * Use case ends
 
-* 4a. A error occur during installation
-  * 4a1. The app informs the user that the installation is unsuccessful and display reasons
+* 4a. An error occur during installation
+  * 4a1. The app informs the user that the installation is unsuccessful and displays reasons
   * Use case ends
 
-**Use case: Delete a person**
+### Use case: Add a student [UC02]
+
+Preconditions
+1. User has installed the app
+
+MSS
+1. User chooses to add a new student.
+2. User enters the required details to create a student.
+3. TutorMate creates the student.
+4. The app shows the user the new student created.
+
+Use Case ends
+
+Extensions
+* 2a. An error parsing the user entered information was detected
```

Comment:
Would it be better if fewer implementation details are included (eg. User did not enter all the required details to create a student.) ?

Reply:
thanks for the suggestion